notifications/smtp/smtp.go

```diff
@@ -0,0 +1,114 @@
+package smtp
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"mime/multipart"
+	"net/mail"
+	"net/smtp"
+	"net/textproto"
+
+	"github.com/vocdoni/saas-backend/notifications"
+)
+
+type SMTPConfig struct {
+	FromName     string
+	FromAddress  string
+	SMTPServer   string
+	SMTPPort     int
+	SMTPUsername string
+	SMTPPassword string
+}
+
+type SMTPEmail struct {
+	config *SMTPConfig
+	auth   smtp.Auth
+}
+
+func (se *SMTPEmail) Init(rawConfig any) error {
+	// parse configuration
+	config, ok := rawConfig.(*SMTPConfig)
```

Comment:
Umm, why rawConfig is any? Is this implementing an interface?

Reply:
yes it is an interface